src/main.rs

```diff
@@ -705,57 +691,6 @@ fn run() -> Result<(), failure::Error> {
             ("", None) => message::warn("kv:bulk expects a subcommand"),
             _ => unreachable!(),
         }
-    } else if let Some(kv_matches) = matches.subcommand_matches("kv:bucket") {
```

Comment:
Need to remove all bucket logic in the Clap::App declaration above as well

Reply:
did i not? lame.

Reply:
Just did for ya!

Reply:
oh man you confused to 💩 out of me. thanks!